wiki/mapping/intermediate-lighting.md

```diff
@@ -17,7 +17,127 @@ You've got your lighting feet wet in [Basic Lighting](./basic-lighting.md) and h
 all... great! Welcome! This page will help you expand on your "vanilla" lighting technique library but as Puds says, now
 it's time to start exploring and experimenting. Have fun with it!
 
-## Off Event Timing
+## Grasping the Group Lighting System
+One of the most versatile aspects of the group lighting system is the
+ability to control individual segments of an object. This is realized
+by splitting an object into sections by filtering and limiting.
+
+![The group info menu](~@images/mapping/groupinfo.png)
+
+The group info menu is in the top-right corner of the official editor.
+To the left of the menu are tabs for settings of individual lanes, as
+well as buttons for adding and deleting lanes.
+
+There are four subsections in the menu: Duration, Filter & Order, Limit
+and Brightness/Rotation Distribution.
+
+### Filter and Order
+Filter & Order is the main mechanism for choosing which segments a lane
+affects.
+
+The segments are first split into chunks, each chunk consisting of
+adjacently-numbered segments. You are able to specify the total number
+of chunks, and the game will group the segments in a way as even as
+possible. For a light with 24 segments, setting the number of chunks
+to 1 would group all segments into one chunk. Setting to 2 instead
+splits the segment into two continuous chunks of 12 lights. Setting to
+3 splits it into chunks of 8, setting to 4 splits it into chunks of 6,
+and so on. Setting to 0 is the same as setting the number of chunks to
+be equal to the total number of segments, that is, each segment resides
+in its own chunk. Filtering, as well as duration and distribution told
+later on, work on chunks.
+
+After the segments are grouped into chunks, filtering is applied to
+choose which chunks are affected by the lane. There are two modes of
+filtering: sections and step/offset. In section mode, the chunks are
+divided into sections as even as possible, in a manner similar to how
+segments are grouped into chunks: section 2 affects the lights right
+after section 1, section 3 affects those right after section 2, and
+so on. The section controlled is given in the Id field.
+
+In step and offset mode, one chunk would be affected by the lane for
+every step. The size of a step is settable. If step is set to 2, then
+the 1st chunk would be controlled by the lane, the 2nd in turn would
+not, the 3rd would, the 4th would not, and so on. If it is 3 instead,
+the lane would control chunks 1, 3, 7, 10 and on. The selection do not
+need to start at the 1st chunk; the offset, titled <q>Light #</q>, is
```

Comment:
Linting error blocks. replace the <q> and </q> with ` to highlight/differentiate or just regular "